Makefile

```diff
@@ -30,3 +30,6 @@ lint: flake8
 
 flake8:
 	flake8 .
+
+test:
+	nosetests assisted-events-scrape
```

Comment:
Not a blocker for this PR, but we should add this as a prow job

Reply:
I'd love to, how can this be done? Do you have a link to any similar PR?

Reply:
this one is a good example: https://github.com/openshift/release/pull/21127/files

Reply:
OK, AFAIU this won't work until this is merged (it will work only for this PR).
I will follow up with CI integration once this gets merged

Reply:
sure, no problem
that's a good plan